test/mitmproxy/addons/test_script.py

```diff
@@ -183,6 +184,20 @@ def test_dupes(self):
                     scripts = ["one", "one"]
                 )
 
+    def test_script_deletion(self):
+        tdir = tutils.test_data.path("mitmproxy/data/addonscripts/")
+        with open(tdir + "/dummy.py", 'w') as f:
```

Comment:
Why not `tutils.test_data.path("mitmproxy/data/addonscripts/dummy.py")`? Am I missing something entirely obvious? 😄 
(also: re-use var below)